### ISSUE TITLE
Remove Google Analytics and other tracking services

### ISSUE THREAD

**Issue by ekaitz-zarraga** [GoteoFoundation/goteo#151]
Hi,

I've wanted to add the widget of my project in my own website and I realized it uses google-analytics.

Google Analytics is known to be a threat to privacy that makes any person that wanted to donate to a project a target. We are not taking care of our people if we use it.

There are many other tools for that which are respectful peoples basic human rights. Matomo, for instance, is a free software alternative (GPL3) that claims to come with privacy by default.

I'm not sure what's the best option to replace Google Analytics, but having it is forcing everyone to be tracked and monitored by a company that is know for threatening basic human rights in many contexts and we are promoting a monopoly to keep controlling our lives.

Please consider the removal of tracking tools. They are unethical.

**Comment by microstudi** [comment]
Yes you're right, in fact we are testing matomo in Goteo right now and we want to make the transition at some point.
The problem we have is the lack of more resources to speed up the task. But we will get there eventually!
Thanks

**Comment by ekaitz-zarraga** [comment]
Also facebook pixel is a VERY aggressive tracking service.

Thanks for thinking about it and trying to change.
